### PR TITLE
Simplify distributed modeling section

### LINKED ISSUE #524: Condense and simplify page about choosing distribution column
## Why are we implementing it? (sales eng)

User testing of the docs has revealed that the section about distributed data modeling has a few key phrases buried inside. Once people see these phrases it answers their questions much better than the surrounding text.

## Communication goals (e.g. detailed howto vs orientation)

Condense the [data modeling](https://docs.citusdata.com/en/v7.0/sharding/data_modeling.html) section to show concrete heuristics right away. For instance two lists side by side with signs your use case is MT vs RT.

### PR DESCRIPTION
**Pull request by begriffs**
Condense the information so people can quickly determine the type of their application for data modeling.

Fixes #524

### CODE REVIEW

**Comment by begriffs** [comment]
I pushed some changes to for your comments on individual lines. Had wanted to finish this whole PR up, but given the impending 7.2 release I better go back to issues tagged for the 7.2 milestone.

**Comment by begriffs** [comment]
@sumedhpathak see how you like this. I cut a lot out of the page, since good coverage exists in other places. Now the page is about determining application type, with best practices for each type and links to the detailed guides.

File: sharding/data_modeling.rst [hunk 1]
```diff
@@ -1,30 +1,37 @@
...
 
-There are characteristics to look for in queries and schemas to determine whether the multi-tenant data model is appropriate. Typical queries in this model relate to a single tenant rather than joining information across tenants. This includes OLTP workloads for serving web clients, and OLAP workloads that serve per-tenant analytical queries. Having dozens or hundreds of tables in your database schema is also an indicator for the multi-tenant data model.
+**Real-Time Analytics**
 
-The second common Citus use case is **real-time analytics**. The choice between the real-time and multi-tenant models depends on the needs of the application. The real-time model allows the database to ingest a large amount of incoming data and summarize it in "human real-time," which means in less than a second. Examples include making dashboards for data from the internet of things, or from web traffic. In this use case applications want massive parallelism, coordinating hundreds of cores for fast results to numerical, statistical, or counting queries.
+  Applications needing massive parallelism, coordinating hundreds of cores for fast results to numerical, statistical, or counting queries.
```
[the remark this answers is not in the document]
Comment (sumedhpathak) [reply sharding/data_modeling.rst]: Re-upping this one. Looks like we changed some of the text for MT, but not for the real-time use-case. We should have, for example, dashboards somewhere here.

File: sharding/data_modeling.rst [hunk 2]
```diff
@@ -36,276 +45,40 @@ The following diagram illustrates co-location in the multi-tenant data model. It
...
-
-    old_length INT,
-    new_length INT
-  );
-
-  SELECT create_distributed_table('wikipedia_editors', 'editor');
-  SELECT create_distributed_table('wikipedia_changes', 'editor');
-
-These tables can be populated by the Wikipedia API, and we can distribute them in Citus by the :code:`editor` column. Notice that this is a text column. Citus' hash distribution uses PostgreSQL hashing which supports a number of data types.
+Real-time queries typically ask for numeric aggregates grouped by date or category. Citus sends these queries to each shard for partial results and assembles the final answer on the coordinator node. Queries run fastest when as many nodes contribute as possible, and when no individual node bottlenecks.
```
Comment (sumedhpathak) [inline sharding/data_modeling.rst]: > Queries run fastest when as many nodes contribute as possible, and when no individual node bottlenecks.

This sentence reads a bit awkwardly. What do we intend to say with 'bottlenecks'?

File: sharding/data_modeling.rst [hunk 3]
```diff
@@ -36,276 +45,40 @@ The following diagram illustrates co-location in the multi-tenant data model. It
...
+* **Choose a column with even distribution.**
+  If you distribute a table on a column skewed to certain common values, then data in the table will tend to accumulate in certain shards. The nodes holding those shards will end up doing more work than other nodes.
 
-Events and Reference Tables
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* **Distribute fact and dimension tables on their common columns.**
+  Your fact table can have only one distribution key. Tables that join on another key will not be co-located with the fact table. Choose one dimension to co-locate based on how frequently it is joined and the size of the joining rows. 
 
-We've already seen how every row in a distributed table is stored on a shard. However for small tables there is a trick to achieve a kind of universal :ref:`co-location <colocation>`. We can choose to place all its rows into a single shard but replicate that shard to every worker node. It introduces storage and update costs of course, but this can be more than counterbalanced by the performance gains of read queries.
+* **Change some dimension tables into reference tables.**
```
Comment (sumedhpathak) [inline sharding/data_modeling.rst]: Link to reference table section here? As I noted above, maybe a brief paragraph earlier on which talks about reference tables might be useful.

Comment (begriffs) [reply sharding/data_modeling.rst]: Link added.

**Pull request merged**